### PR TITLE
Render measure in print.

### PR DESCRIPTION
**Pull request by tonio**
Transition from HTML tooltips to ol styles.

The sketchStyle class ngeo methods to get measure, store it in feature property on `measureend`, & final style use it, because text in tooltip is not available anymore.
Feature is stored & readded to layer when active tool changes.

A `hidden` property is added to measure layer, which prevent it to be displayed in layer manager.

Needs [small ngeo tweak](https://github.com/camptocamp/ngeo/commit/b178a9e7451a4b660c728fedcf015e9f9854cd50).

(last minute check show me azimut isn't drawn in print… is `Circle` supported in MF print?)

### CODE REVIEW

**Comment by gberaudo** [comment]
@tonio, I would say circles are supposed to work. They are mentioned in the doc: https://mapfish.github.io/mapfish-print-doc/styles.html#stylev2

**Comment by gberaudo** [comment]
Reset does not seem to work for lenghtes.

File: geoportal/geoportailv3_geoportal/static-ngeo/js/measure/MeasureController.js [hunk 1]
```diff
@@ -104,14 +144,20 @@ const exports = function($scope, $q, $http, $compile, gettext,
       fill: new olStyleFill({
         color: 'rgba(255, 204, 51, 0.3)'
       })
+    }),
+    text: new olStyleText({
+      font: 'bold 12px Calibri,sans-serif',
+      fill: new olStyleFill({
+        color: 'rgb(0 0 0 / 100)'
```
Comment (gberaudo) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/measure/MeasureController.js:183]: I don't know this notation, what does it do?

File: geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js [hunk 2]
```diff
@@ -798,6 +798,13 @@ const MainController = function(
    */
   this['selectedLayers'] = [];
 
+  /**
+   * @type {function}
+   */
+  this['selectedLayersLength'] = function() {
```
Comment (gberaudo) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js:804]: Brackets were used for closure-compiler. It is not necessary anymore. Better not to use them in new code.

File: geoportal/geoportailv3_geoportal/static-ngeo/js/measure/MeasureController.js (hunk 1 above)
Comment (tonio) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/measure/MeasureController.js:183]: Should have been `rgb(0 0 0 / 100%)` (or even w/o the `/ 100%`, the default), see https://mobile.twitter.com/mathias/status/1253242715304857601 for ref.

File: geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js (hunk 2 above)
Comment (tonio) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js:804]: I wondered but keep brackets for coherence with the rest of the file, I can change if needed.

**Comment by tonio** [comment]
@gberaudo this one is ready from my side. Is it good to merge for you?

**Pull request merged**